src/api/goods.js

```diff
@@ -6,6 +6,10 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => getAll()
+  .then(data => data.filter((good, index) => index < 5));
 
-export const getRedGoods = () => {};
+export const getRedGoods = () => getAll()
+  .then(data => [...data]
+    .sort((goodA, goodB) => goodA.name.localeCompare(goodB.name))
+    .filter(good => good.color === 'red'));
```

Comment:
You sorted different set of goods than in the task.
`getRedGoods` must only filter red goods, 
`get5First` should first sort goods by name and then return first 5 of them.